.gitmodules

```diff
@@ -1,2 +1,2 @@
 [submodule "gazebo/ardupilot_gazebo"]
-	url = git@github.com:SwiftGust/ardupilot_gazebo.git
+	url = git@github.com/SwiftGust/ardupilot_gazebo.git
```

Comment:
@johncarl81 you are mixing ssh and https
So it should either be `git@github.com:SwiftGust/ardupilot_gazebo.git`
or `https://github.com/SwiftGust/ardupilot_gazebo.git`

Reply:
Weird, this was from a rebase I did.  I'll revert.